gruntfile.js

```diff
@@ -145,6 +152,6 @@ module.exports = function(grunt) {
         grunt.task.run('copy');
     });
 
-    grunt.registerTask('serve', ['showcase', 'less:development', 'connect:livereload', 'openfin:serve', 'watch']);
+    grunt.registerTask('serve', ['showcase', 'less:development', 'connect:livereload', 'download', 'openfin:serve', 'watch']);
     grunt.registerTask('build', ['openfin:build', 'showcase']);
```

Comment:
Does this mean that everytime we run `serve` we'll be connecting to a remote server to download the zip?

Perhaps it would be better to only do that when doing a build or release?